src/haz3lweb/Keyboard.re

```diff
@@ -43,7 +43,7 @@ let handle_key_event = (k: Key.t, ~model: Model.t): list(Update.t) => {
     | "F3" => toggle(Log.debug_update)
     | "F4" => toggle(Log.debug_keystroke)
     | "F5" => toggle(Log.debug_zipper)
-    | "F6" => []
+    | "F6" => now(Jump(BindingSiteOfIndicatedVar)) // Figure out actual keybinding
```

Comment:
Thoughts?

Reply:
we tend to use vscode as our source of defaults, so let's say F12? (plain F12 not shift F12)